docs/community/com-config/com-config-fb-scaling.md

```diff
@@ -7,28 +7,38 @@ grand_parent: Community
 
 ## How do I scale FeatureBase Community?
 
+This page explains the process to scale a FeatureBase cluster.
+
 {: .note}
-You can also [Resize a FeatureBase Cluster](/docs/community/com-config/old-resize-cluster)
+>You can also scale FeatureBase by starting or stopping Ingester instances.
+>* [Learn about FeatureBase ingest](/docs/community/com-ingest/com-ingest-manage)
 
-Generally, scaling is handled by starting or stopping Ingester instances.
+## Before you begin
 
-## Scale FeatureBase up
+{% include /com-install/com-install-before-begin.md %}
+* [Learn about FeatureBase Clusters](/docs/community/com-config/old-resize-cluster)
 
-To scale FeatureBase up, add a node to the cluster and provision it similarly to the others (with its own bind), and make sure it has gossip seeds which are already in the cluster. It will automatically join the cluster and FeatureBase will initiate a resize operation to balance the data in the cluster.
+{: .important}
+FeatureBase will deny any request to access index data while it is in the `resize` state. `/status` and `/version` requests will still work.
 
-## Scale FeatureBase down
+## How do I scale FeatureBase up?
 
-When these steps are complete, FeatureBase will:
-* put the cluster into the `resizing` state
-* move data to make all nodes contain the correct shards given the cluster replication factor
+This process results in FeatureBase initiating a `resize` operation to balance the data in the cluster
 
-{: .important}
-FeatureBase will deny any request to access index data while in the `resize` state. `/status` and `/version` requests will still work.
+1. Add a node to the cluster
+2. Provision the node so it is similar to the others but with its own `bind` value
+3. Make sure `gossip seeds` which are already setup in the cluster.
+
+## How do I scale FeatureBase down?
+
+This process results in FeatureBase:
```

Comment:
Again this info isn't just about scaling down. I think it should be in its own section

Reply:
rejigged as the intro to the page